app/src/main/java/com/x8bit/bitwarden/data/autofill/accessibility/util/BrowserUtil.kt

```diff
@@ -123,6 +123,7 @@ private val ACCESSIBILITY_SUPPORTED_BROWSERS = listOf(
     Browser(packageName = "org.chromium.chrome", urlFieldId = "url_bar"),
     Browser(packageName = "org.codeaurora.swe.browser", urlFieldId = "url_bar"),
     Browser(packageName = "org.cromite.cromite", urlFieldId = "url_bar"),
+    Browser(packageName = "foundation.e.browser", urlFieldId = "url_bar"),
```

Comment:
Can this be moved to maintain alphabetic ordering?

Reply:
Done